### PR TITLE
[ZEPPELIN-1585] Testcase for PySparkInterpreter.

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?

This PR adds PySparkInterpreter testcase.
### What type of PR is it?

Improvement
### Questions:
- Does the licenses files need update? no
- Is there breaking changes for older versions? no
- Does this needs documentation? no

### CODE REVIEW

**Comment by minahlee** [comment]
Zeppelin doesn't support pyspark with Spark 1.1.x. You might need to add `if` clasue to skip pyspark test for Spark 1.1.x.

**Comment by astroshim** [comment]
@minahlee I fixed it. 
Thank you for proper comments.

**Comment by minahlee** [comment]
It would be nice if I can see all CI profiles pass.

File: spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java [hunk 1]
```diff
@@ -165,6 +160,15 @@ public void open() {
     }
   }
 
+  private Map setupPySparkEnv() throws IOException{
+    Map env = EnvironmentUtils.getProcEnvironment();
+    if (!env.containsKey("PYTHONPATH")) {
+      SparkConf conf = getSparkConf();
+      env.put("PYTHONPATH", conf.get("spark.files").replaceAll(",", ":"));
```
Comment (felixcheung) [inline spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java]: should this be `spark.submit.pyFiles` (or both `spark.files` and `spark.submit.pyFiles`)?
http://spark.apache.org/docs/latest/configuration.html#runtime-environment

**Comment by astroshim** [comment]
rebuild-CI

**Comment by astroshim** [comment]
@felixcheung I fixed it and CI has passed. Thank you very much!

**Comment by jongyoul** [comment]
LGTM

**Comment by minahlee** [comment]
Merging if there no more discussion

File: spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java [hunk 2]
```diff
@@ -45,13 +45,8 @@
 import org.apache.spark.SparkConf;
 import org.apache.spark.api.java.JavaSparkContext;
 import org.apache.spark.sql.SQLContext;
-import org.apache.zeppelin.interpreter.Interpreter;
-import org.apache.zeppelin.interpreter.InterpreterContext;
-import org.apache.zeppelin.interpreter.InterpreterException;
-import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.apache.zeppelin.interpreter.*;
```
Comment (astroshim) [inline spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java]: okay thanks.

**Pull request closed without merging**